cid-redirects.json

```diff
@@ -1404,6 +1404,7 @@
   "/cid/1958": "/docs/integrations/amazon-aws/security-hub",
   "/cid/1959": "/docs/integrations/containers-orchestration/vmware",
   "/cid/1960": "/docs/integrations/saas-cloud/pagerduty-v2",
+  "/cid/2122": "/docs/integrations/saas-cloud/gmail-tracelogs",
```

Comment:
This will work once you merge https://github.com/SumoLogic/sumologic-documentation/pull/1091

Reply:
Thanks, @kimsauce:  I just merged #1091, but this is still failing.

Reply:
@kimsauce Thanks Kim, it is merged now.

Reply:
You have to push this button first to update this build. It succeeded after I did that:

<img width="803" alt="Screen Shot 2022-12-02 at 10 54 35 AM" src="https://user-images.githubusercontent.com/56411016/205368943-367659a4-4f08-43b7-9dee-315fcbb6a232.png">

Reply:
Yeah I saw that. Thank you so much @kimsauce.